### PR TITLE
feature/textarea

### PR DESCRIPTION
**Pull request by MKaHead**
- implementation of textarea
- fixes and changes for input-text
- bump lit element lit html versions

### CODE REVIEW

**Comment by LucaMele** [comment]
Beim submit geht im IE die demo Kaputt

Bildschirmfoto 2019-07-02 um 11 24 14

File: src/components/10-atoms/input-text/README.md [hunk 1]
```diff
@@ -84,9 +84,9 @@ The boolean attribute `required` visualizes an element that must obligatorily be
 
 String-valued `placeholder`specifies the placeholder text shown when the element is empty.
 
-### valid
+### invalid
 
-The boolean attribute `valid` serves to indicate the validity of the element (default: `true`).
+The boolean attribute `invalid` serves to indicate the validity of the element (default: `false`). if `false` it sets the element into a visual error state.
 
 ### validation
```
Comment (LucaMele) [inline src/components/10-atoms/input-text/README.md]: validation
The boolean attribute validation shows an animated checkmark to the right of the input-text, if valid is true (default: false). 

-> Better name might be `animation`

**Comment by LucaMele** [comment]
This demo is confusing:

Bildschirmfoto 2019-07-02 um 11 34 41

**Comment by LucaMele** [comment]
Cant see a placeholder in the with placeholder example:

Bildschirmfoto 2019-07-02 um 11 35 41

File: src/components/10-atoms/input-text/README.md (hunk 1 above)
Comment (LucaMele) [reply src/components/10-atoms/input-text/README.md]: same here:
Bildschirmfoto 2019-07-02 um 11 43 25

**Comment by LucaMele** [comment]
Error messages and placeholder messages should not go over the component itself (maybe double check with designer if is intended)

Bildschirmfoto 2019-07-02 um 11 45 38

**Comment by LucaMele** [comment]
There is no information on how to use the character counter in the readme. Please add it

File: src/components/10-atoms/input-text/README.md (hunk 1 above)
Comment (MKaHead) [reply src/components/10-atoms/input-text/README.md]: fixes in 2238ff5

**Pull request merged**